src/System.Net.Requests/tests/RequestStreamTest.cs

```diff
@@ -11,11 +11,18 @@
 
 namespace System.Net.Tests
 {
-    [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Tests hang on .NET Framework")]
     public class RequestStreamTest
     {
         readonly byte[] buffer = new byte[1];
 
+        static RequestStreamTest()
+        {
+            // On .NET Framework, the default limit for connections/server is very low (2).
+            // On .NET Core, the default limit is higher. Since these tests run in parallel,
+            // the limit needs to be increased to avoid timeouts when running the tests.
+            ServicePointManager.DefaultConnectionLimit = int.MaxValue;
```

Comment:
So talked offline to @ViktorHofer and yes there is a leak because we never dispose the RequestStream, so we should then dispose the GetRequestStream so that the connection is closed.